src/Integration/Resources/Strings.resx

```diff
@@ -723,4 +719,12 @@ This is the general format to use when writing errors to output window or when t
   <data name="Bind_NuGetAnalyzersNoLongerInstalled" xml:space="preserve">
     <value>INFO: SonarLint connected mode no longer installs the analyzers NuGet packages. The analyzers embedded in the VSIX are used instead.</value>
   </data>
+  <data name="Notifications_NotSupported" xml:space="preserve">
+    <value>Notifications are not supported on this version of SonarQube</value>
+    <comment>Output window message</comment>
+  </data>
+  <data name="OnlySupportedPluginHasNoProjectInSolution" xml:space="preserve">
```

Comment:
This message already existed  It has been relocated from line 649 - not sure why,